homeassistant/components/alexa/handlers.py

```diff
@@ -406,7 +407,7 @@ async def async_api_lock(hass, config, directive, context):
 
     response = directive.response()
     response.add_context_property(
-        {"name": "lockState", "namespace": "Alexa.LockController", "value": "LOCKED"}
```

Comment:
Does it make sense to use (partly) constants for external responses like this?
Personally, I'm not really sure about that.